clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -145,6 +151,18 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Get all vips in a region.
+   * @param region
+   * @return
+     */
+  List<? extends Vip> listVips(final String region) {
+    List<? extends Vip> vips = handleRequest {
```

Comment:
Can you remove this assignment and return statement?  Let's follow the existing pattern.